configuration/homebuilder.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: documentation
+title: Home Builder
+source: https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.ui.homebuilder/USAGE.md
+---
+
+{% include base.html %}
+
+<!-- Attention authors: Do not edit directly. Please add your changes to the appropriate source repository -->
+
+# Home Builder
+
+Boilerplate for the [Items](http://docs.openhab.org/configuration/items.html), [sitemap](http://docs.openhab.org/configuration/sitemaps.html) files and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard.
```

Comment:
Should I change it to `{{base}}` in the original file?

Reply:
I have checked the openhabian repo to evaluate this and there are also absolute urls in the readme.
I think it should stay as it is.

Reply:
Note that this is merely a copied version from https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.ui.homebuilder/USAGE.md, so there is no need to complain about it on this PR :-)

So, yes, absolute urls are imho ok (because that page also exists on github elsewhere), but all of them should actually use https and not http. I'll leave a comment on PR in openhab-core...

Reply:
External articles should work with absolute URLs, not that it matters too much...

https, indeed. Should be part of #103 / #566 which I didn't drive further yet